cookbook/docs/conf.py

```diff
@@ -447,6 +449,7 @@ def hide_example_page(file_handler):
     "scipy": ("https://docs.scipy.org/doc/scipy/reference", None),
     "matplotlib": ("https://matplotlib.org", None),
     "flytekit": ("https://flyte.readthedocs.io/projects/flytekit/en/latest/", None),
+    "flytekitplugins": ("https://docs.flyte.org/projects/flytekit/en/latest/", None),
```

Comment:
I could be wrong here, but I believe the `flytekit` line right above this one will already give us access to the plugins reference.

Reply:
No, it isn't. [Current version](https://docs.flyte.org/projects/cookbook/en/latest/auto/integrations/flytekit_plugins/greatexpectations/task_example.html#simple-task) vs. [PR-generated docs](https://flyte--861.org.readthedocs.build/projects/cookbook/en/861/auto/integrations/flytekit_plugins/greatexpectations/task_example.html#simple-task) (hover on `GreatExpectationsTask`)

Reply:
huh, do any of the flytekit references show up? 

both https://flyte.readthedocs.io/projects/flytekit/en/latest/ and https://docs.flyte.org/projects/flytekit/en/latest/ resolve to the same url.

Can we try using  single reference key `flytekit` to the [correct](https://docs.flyte.org/projects/flytekit/en/latest/) url? AFAIK the key `flytekit` or `flytekitplugins` only matters with cross referencing using `:ref:` roles

Reply:
nvm, it does look like `flytekitplugins` has to be in the intersphinx config for it to automatically get linked in source code.